docs/source/deployment/airflow.md

```diff
@@ -0,0 +1,173 @@
+# Apache Airflow
+
+Apache Airflow is a popular open-source workflow management platform. It is a suitable engine to orchestrate and execute a pipeline authored with Kedro because workflows in Airflow are modelled and organised as [DAGs](https://en.wikipedia.org/wiki/Directed_acyclic_graph).
+
+## How to run a Kedro pipeline on Apache Airflow with Astronomer
+
```

Comment:
For users who wish to deploy on using Atronomer  this tutorial is clear. For users that have a different Airflow provider, it should be mentioned which steps are not relevant and that Kedro Airflow is not specific to astronomer.

In the logical flow (reading top to bottom), this information should be placed above Astronomer specific instructions